opentelemetry-etw-logs/CHANGELOG.md

```diff
@@ -22,6 +22,8 @@
 
 - Added support for TraceId,SpanId
 
+- Added support for populating cloud `role` and `roleInstance` from Resource's `service.name` and `service.instance.id` attributes respectively.
```

Comment:
I suggest to do refactoring on own PR, separate from adding features. It is a lot easier to review PR that only does one thing.